### PR TITLE
Improve loading spinner

### PR DESCRIPTION
**Pull request by benedikt-richter**
### Summary of changes

Instead of opening a new window to display a loading spinner, the loading spinner is displayed in the same window. 

### Context and reason for change

Before:
Screenshot 2022-09-28 at 21 06 55

After:
Screenshot 2022-09-28 at 21 04 20

Fix: #966

### How can the changes be tested

Test that the new spinner is shown while files are opened and during file export.

### CODE REVIEW

**Comment by benedikt-richter** [comment]
With the most recent change the loading spinner does also work properly for file exports. Before, for most of the duration of the export there was no spinner shown but the app was just frozen.

File: public/loading.html [hunk 1]
```diff
@@ -1,43 +0,0 @@
-<!DOCTYPE html>
```
Comment (antonbauhofer) [inline public/loading.html:1]: Where is this file coming from / what is it's purpose?

**Review by antonbauhofer — APPROVED** [review]
Well done. The new spinner looks much better than before!
The code looks good and it works as expected.

File: public/loading.html (hunk 1 above)
Comment (benedikt-richter) [reply public/loading.html:1]: that's the old spinner

**Pull request merged**